department-of-veterans-affairs/caseflow
# Loading spinners

Connects #5824 

### Description
This PR adds the loading spinner for the VACOLS upload

### Testing Plan
1. Create a schedule period through the UI
1. Confirm that the success page displays the correct loading message
